.github/workflows/velox_be.yml

```diff
@@ -38,91 +38,92 @@ on:
       - 'substrait/substrait-spark/**'
 
 env:
-  HTTP_PROXY_HOST: 10.240.252.16
+  HTTP_PROXY_HOST: proxy-shz.intel.com
   HTTP_PROXY_PORT: 911
-  DOCKER_REGISTRY_HOST: 10.1.0.25
-  DOCKER_REGISTRY_PORT: 5000
+  PATH_TO_GLUTEN_TE: ./tools/gluten-te
+  DOCKER_PUSH_REGISTRY: 10.1.0.25:5000
+  DOCKER_CACHE_REGISTRY: 10.1.0.25:5000
+  PREVIOUS_DOCKER_PUSH_REGISTRY: 10.0.2.4:5000
 
 concurrency:
   group: ${{ github.repository }}-${{ github.head_ref || github.sha }}-${{ github.workflow }}
   cancel-in-progress: true
 
 jobs:
-
   ubuntu2004-test-spark32:
     runs-on: velox-self-hosted
+    env:
+      OS_IMAGE_NAME: ubuntu
+      OS_IMAGE_TAG: 20.04
     steps:
-      - uses: actions/checkout@v4
```

Comment:
Sorry, this may still be required... Since the script files are stored in Gluten's code base?